### PR TITLE
Login and signup

### PR DESCRIPTION
**Pull request by HeyNitin**
Added login and signup 

image
image

### CODE REVIEW

File: src/pages/explore/explore.tsx [hunk 1]
```diff
@@ -1,17 +1,57 @@
...
 import { useVideo } from "contexts/videoContext/videoContext";
+import axios from "axios";
+import { showToast } from "components/toast/toast";
+import { useEffect } from "react";
+import { video } from "contexts/videoContext/videoContext.type";
 
 const Explore = (): JSX.Element => {
 	const { state, dispatch } = useVideo();
 
+	const applyFilteres = (product: video) => {
```
Comment (MarkVed17) [inline src/pages/explore/explore.tsx:13]: Filters*

File: src/services/validatorServices.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const nameFormat = /^[a-zA-Z]{2,25}( [a-zA-Z]{2,25})+$/;
```
Comment (MarkVed17) [inline src/services/validatorServices.tsx:1]: I liked that you are handling edge cases using regex.

**Review by MarkVed17 — APPROVED** [review]
Good work adding types in your code.
Good folder structure 👌🏼
Well-organised code 👍🏼
Looks good to proceed!

File: src/pages/explore/explore.tsx (hunk 1 above)
Comment (HeyNitin) [reply src/pages/explore/explore.tsx:13]: Thanks for pointing it out, I've changed it in local, will push with new branch

**Pull request merged**